.github/actions/build-llvm/action.yml

```diff
@@ -16,7 +16,7 @@ runs:
     - name: Clone LLVM framework
       shell: ${{ runner.os == 'Windows' && 'msys2 {0}' || 'bash' }}
       run: |
-        cargo install compiler-llvm-builder
+        cargo install compiler-llvm-builder --git https://github.com/matter-labs/era-compiler-llvm-builder --branch aba-remove-wget-tar-requirements
```

Comment:
TODO: restore this line when https://github.com/matter-labs/era-compiler-llvm-builder/pull/26 merged